src/Core/Users/UserLogin.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+class UserLogin implements UserInterface {
```

Comment:
I don't see any declarations for class variables that you are using in methods. Please add them.